usaspending_api/broker/management/sql/update_tables.sql

```diff
@@ -5,6 +5,7 @@ ALTER TABLE legal_entity RENAME TO legal_entity_old;
 ALTER TABLE transaction_normalized RENAME TO transaction_normalized_old;
 ALTER TABLE awards RENAME TO awards_old;
 ALTER TABLE references_legalentityofficers RENAME TO references_legalentityofficers_old;
+ALTER TABLE duns RENAME TO duns_old;
```

Comment:
How often does `update_tables.sql`  run? Is it automatically or manually started?

Reply:
I'm not quite sure. When developing I was referred to this file but looking at it, it might be only run for initial loading of the website data before the nightly loader updates. @nmonga91 can you confirm?